test/ScanDependencies/module_deps_cache_reuse.swift

```diff
@@ -114,7 +111,7 @@ import SubE
 // CHECK-NEXT: "-frontend"
 // CHECK-NEXT: "-only-use-extra-clang-opts"
 // CHECK-NEXT: "-Xcc"
-// CHECK-NEXT: "clang"
+// CHECK-NEXT: "{{.*}}clang"
```

Comment:
Im not sure if this is the right thing to do - we really want to ensure that the correct clang is invoked (normally `BUILD_DIR/clang`).